File: phonebook_server/phonebook_server/Dtos/GetPhonebookDto.cs
using phonebook_server.Models;

namespace phonebook_server.Dtos
{
    public class GetPhonebookDto
    {
        public int Id { get; set; }
        public string Email { get; set; }
        public string Name { get; set; }
        public string Surname { get; set; }
        public string HomePhoneNumber { get; set; }
        public string CellPhoneNumber { get; set; }
        public District District { get; set; }
    }
}

File: phonebook-client/src/app/dtos/GetPhonebookDto.ts
export class GetPhonebookDto {
  public id: number;
  public email: string;
  public name: string;
  public surname: string;
  public homePhoneNumber: string;
  public cellPhoneNumber: string;
  public district: District;

  constructor() {}
}

export class District {
  public id: number;
  public postCode: string;
  public name: string;
}
